src/data/valid/Database-neon_Biosample_to_DataObject_NEON.yaml

```diff
@@ -97,25 +97,6 @@ biosample_set:
       term:
         type: nmdc:OntologyClass
         id: ENVO:00005792
-data_object_set:
```

Comment:
Why did these lines get deleted?  This is one of the few/only examples where we have ref integrity w/in the example file so I'd like to keep intact.

Reply:
NM, i see the section just got moved.